cluster/addons/fluentd-gcp/fluentd-gcp-image/Dockerfile

```diff
@@ -30,17 +30,21 @@ ENV DO_NOT_INSTALL_CATCH_ALL_CONFIG true
 
 RUN apt-get -q update && \
     apt-get install -y curl && \
+    apt-get install -y gcc && \
+    apt-get install -y make && \
     apt-get clean && \
```

Comment:
@a-robinson fyi, Found another tip to cleanup files in addition to "apt-get clean" in this PR:
https://github.com/kubernetes/kubernetes/pull/27901/commits/8b20890bed4145887f0e5056f8a250260be11aa9

`rm -rf /var/lib/apt/lists/*`